{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -21,6 +21,12 @@ requirements:
 	$(PYTHON_INTERPRETER) -m pip install -U pip
 	$(PYTHON_INTERPRETER) -m pip install -r requirements.txt
 	{% endif -%}
+	{% elif "pyproject.toml" == cookiecutter.dependency_file -%}
+	{% if "uv" == cookiecutter.environment_manager -%}
+	uv sync
```

Comment:
`uv sync` is nicer than `uv pip install -e .`. You get a lockfile, and it makes sure the environment matches the declared dependencies.